quirk_echo_enclave_app/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "quirk_echo_enclave_app"
+version = "0.1.0"
+authors = ["Dzmitry Huba <huba@google.com>"]
+edition = "2021"
```

Comment:
Is this supposed to be the year?

Reply:
Yes, however this is not the current year, but rather the edition of the tool chain.